src/QMCTools/ppconvert/README

```diff
@@ -12,6 +12,12 @@
 
 ppconvert supports conversion of pseudopotentials between different formats
 
-ppconvert is currently not built automatically, but only requires a C++ compiler and BLAS/LAPACK
+ppconvert is currently not built automatically, but can be built on limited platforms
+(Centos7 intel 17.0.0 as of now) it may work on other platforms. Clang and GCC support
+with MKL are planned.
 
-Edit the Makefile to include an appropriate compiler and BLAS source. Examples using MKL are included.
+cmake -DBUILD_PPCONVERT
```

Comment:
cmake -DBUILD_PPCONVERT=1
If you don't assign a value, cmake will complain. Because QMCPACK doesn't use on/off, use 1.